app/gui/src/profile_workflow.rs

```diff
@@ -40,20 +43,43 @@ pub async fn entry_point_profile() {
 // === Workflow definitions ===
 // ============================
 
-#[profile(Objective)]
-async fn profile_add_node() {
-    let test = Fixture::setup_new_project().await;
-    let graph_editor = test.graph_editor();
-    let initial_nodes = graph_editor.nodes().all.len();
-    let expect_node_added = graph_editor.node_added.next_event();
-    graph_editor.add_node();
-    let _ = expect_node_added.expect();
-    let added_nodes = 1;
-    assert_eq!(graph_editor.nodes().all.len(), initial_nodes + added_nodes);
+async fn profile_create_node() {
+    let test = Fixture::create_project().await;
+    let _profiler = profiler::start_objective!(profiler::APP_LIFETIME, "@highlight");
```

Comment:
Can `@highlight` be moved to some constant?

Reply:
Profiler labels must be literals to enable packing file/line info into the string, which has a large impact on runtime profiling data size.